### PR TITLE
Add automated postflight check code into install folder.

### PR DESCRIPTION
**Pull request by LiuYuWei**
** PR checklist **

- [x] Ensure you have added or ran the appropriate tests for your PR.
- [x] DCO signed

**What type of PR is this?**

feature.

**What this PR does / why we need it**:

Do the postflight check automatically.

**Which issue(s) this PR fixes**:

sc-24603

**Special notes for your reviewer**:

Please see the README file first to know how to use this code.

**Does this PR introduce a user-facing change?**:

```release-note
- Build up auto post-flight check method.
```

### CODE REVIEW

File: install/postflight-check/main.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+"""This file is main file."""
+import time
+import argparse
+from src.setup_logger import logger
+from src import PostflightCheck
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description='my description')
```
Comment (jimmyliao) [inline install/postflight-check/main.py:9]: rename to meaningful description.

File: install/postflight-check/docker/image.yaml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+postflight:
+  postflight: simon3458/primehub-postflight-check:v0.0.4
```
Comment (jimmyliao) [inline install/postflight-check/docker/image.yaml:2]: nit: could put the image within company wise, or w/o docker in the future.

File: install/postflight-check/main.py (hunk 1 above)
Comment (LiuYuWei) [reply install/postflight-check/main.py]: Change it. Done.

File: install/postflight-check/docker/image.yaml (hunk 2 above)
Comment (LiuYuWei) [reply install/postflight-check/docker/image.yaml]: Change it. Done.

**Review by jimmyliao — APPROVED** [review]

**Pull request merged**